domains/enginecreations.json

```diff
@@ -0,0 +1,12 @@
+{
+  "description": "Engine Creations(TM) main pages",
+  "repo": "https://github.com/akk1to/enginecreations.github.io",
+  "owner": {
+    "username": "Engine Creations(TM)",
+    "email": "contact@enginecreations.is-a.dev",
```

Comment:
This file has no MX record so it can not receive mails

Reply:
resolved the change